packages/inputs/src/Table.js

```diff
@@ -275,6 +275,17 @@ export class Table extends Input {
     }
   }
 
+  reset() {
+    this.currentRow = -1;
+    
+    if (this.sortHeader) {
```

Comment:
Reset should only reset interface state related to any (now removed) selection clauses.